client/blocks/reader-featured-video/index.jsx

```diff
@@ -61,14 +65,11 @@ class ReaderFeaturedVideo extends React.Component {
 	throttledUpdateVideoSize = throttle( this.updateVideoSize, 100 )
 
 	handleThumbnailClick = ( e ) => {
-		e.preventDefault();
-		this.props.onThumbnailClick();
-
-		if ( ! this.props.allowPlaying ) {
-			return false;
+		if ( this.props.allowPlaying ) {
```

Comment:
So the way I have it marked now that I think solved it:
1. If `allowPlaying` is true _then_ call `this.props.onThumbnailClick` for outputting stats
2. if `allowPlaying` is false, then we are really just displaying an image.  I pass through an `href` so that it works like a usual link